packages/test-utils/src/matches.js

```diff
@@ -4,10 +4,20 @@ import {
   FUNCTIONAL_OPTIONS
 } from 'shared/consts'
 import { isConstructor } from 'shared/validators'
+import { capitalize, camelize } from 'shared/util'
 
-export function vmMatchesName(vm, name) {
+function vmMatchesName(vm, name) {
+  console.log(name)
```

Comment:
some debugging stuff over here! ;)

Reply:
Do we want a lint rule for no console log? We do this at work. Same for debugger statements

Reply:
I think its OK to have one yeah :) With the new linting on commit, it will warn you.